src/views/FoodOrder.vue

```diff
@@ -55,18 +57,24 @@ export default class FoodOrder extends Vue {
     require("@/assets/images/FoodOrderImages/Sushi.png")
   ];
   /* eslint-enable */
+
+  private passToNextStep(): void {
+    this.stepperStatus = (+this.stepperStatus + 1).toString();
+  }
 }
 </script>
 
 
 <style scoped>
-
+.stepper {
+  width: 70%;
+}
 .v-card {
     display: flex;
     justify-content: center;
 }
 .stepperImagesStyle {
-  width: 16vw;
+  width: 25%;
```

Comment:
למה אחוזים במקום vw?

Reply:
Because there are 4 images 25% assure that all the space will be used